enos/modules/vault_artifactory_artifact/locals.tf

```diff
@@ -5,23 +5,23 @@ locals {
     amd64 = {
       ubuntu = {
         "oss"     = "-1_amd64.deb"
-        "ent"     = "+ent-1_amd64.deb"
-        "ent.hsm" = "+ent-1_amd64.deb"
+        "ent"     = "-1_amd64.deb"
```

Comment:
Good stuff... thanks for that. This was not a fun piece of code to write and I'm glad you're fixing this issue.

Reply:
What a nightmare this local is. Kudos.